### PR TITLE
Added hold to show role functionality to drawer

### PR DESCRIPTION
**Pull request by ThyHG**
Also no longer uses the store to keep status of role drawer being open

### CODE REVIEW

**Review by ThyHG — COMMENTED** [review]
waar komt dit terecht?

File: secrethitler.web/src/components/player/identity-screen/index.js [hunk 1]
```diff
@@ -5,7 +5,25 @@ import {roleStrings} from '../../../helpers/roleTranslator';
...
 class IdentityScreen extends Component {
+	constructor() {
+		super()
+		this.handleButtonPress = this.handleButtonPress.bind(this)
+		this.handleButtonRelease = this.handleButtonRelease.bind(this);
+		this.state = {
+			screenOpen: false
+		}
+	}
+	handleButtonPress () {
```
Comment (ThyHG) [inline secrethitler.web/src/components/player/identity-screen/index.js]:
```suggestion
	handleButtonPress() {
```

File: secrethitler.web/src/components/player/identity-screen/index.js [hunk 2]
```diff
@@ -5,7 +5,25 @@ import {roleStrings} from '../../../helpers/roleTranslator';
...
+		this.setState({
+			screenOpen: true
+		})
+	}
 
+	handleButtonRelease () {
```
Comment (ThyHG) [inline secrethitler.web/src/components/player/identity-screen/index.js]:
```suggestion
	handleButtonRelease() {
```

File: secrethitler.web/src/components/player/identity-screen/index.js [hunk 3]
```diff
@@ -19,9 +37,14 @@ class IdentityScreen extends Component {
 			 bodyText = <p>Here you will find your secret identity when the game has started</p>;
 		 }
 
-		const isOpened = this.props.player.idScreenOpen === true ? 'opened' : 'closed';
+		const isOpened = this.state.screenOpen === true ? 'opened' : 'closed';
```
[the remark this answers is not in the document]
Comment (ThyHG) [reply secrethitler.web/src/components/player/identity-screen/index.js]:
```suggestion
		const isOpened = this.state.screenOpen ? 'opened' : 'closed';
```

**Review by Pineapples — APPROVED** [review]

**Pull request merged**